modules/brew/brew-bash-completions.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/usr/bin/env bash
```

switch to bash shebang

this script seems to have multiple bashisms that shellcheck errors on thanks to the wrong shebang